### PR TITLE
Server checkbox tooltip

### PR DESCRIPTION
**Pull request by Xpirix**
Fix for #487 

image

### CODE REVIEW

File: qgis-app/plugins/models.py [hunk 1]
```diff
@@ -478,7 +478,16 @@ class Plugin(models.Model):
...
 
     # True if the plugin has a server interface
-    server = models.BooleanField(_("Server"), default=False, db_index=True)
+    server = models.BooleanField(
+        _("Server"), 
+        default=False, 
+        db_index=True,
+        help_text=_(
+            "A server plugin is a plugin which can run on QGIS Server,"
+            " by having a entrypoint <b>serverClassFactory</b>, see the"
```
Comment (Gustry) [inline qgis-app/plugins/models.py:487]:
```suggestion
            " by having a entrypoint <pre>serverClassFactory</pre>, see the"
```

**Review by Gustry — APPROVED** [review]
LGTM thanks, it's indeed better for the UX :)

**Comment by Gustry** [comment]
Sorry, you don't prefer `<pre>` instead of `<b>` ? It find it better for computing languages ;-)

**Comment by Xpirix** [comment]
Oops, I'm really sorry. I missed your suggestion; I will fix it right away.

**Comment by Xpirix** [comment]
I've applied it with a new PR at #492

**Pull request merged**